rome_app/templates/rome_templates/page_detail.html

```diff
@@ -119,48 +146,54 @@
 {% endblock %}
 
 {% block content %}
-    <br />
 
-<div id = "bookinfo">
-    {% if book_mode %}
-    <ul> <span style="color:#D64833">Book Information</span> <ul/><br />
-        {% endif %}
-        {% if print_mode %}
-        <ul> <span style="color:#D64833">Print Information</span> :<ul/><br />
-            {% endif %}
-            <ul><b><i>{{ title }}</i>, {{date}}</b></ul>
-            <ul>{{ authors }}</ul>
-            {% if book_mode %}
-            <br>
-            <a href="{{ studio_url }}" target="_blank"> [Library View]</a><span style="float:right;"><ul><b>Brown ID #: </b>{{ book_id }}  </ul></span>
-
-            {% for note in book.note %}
-            {% if "Vincent" in note %}
-            <li>From the collection of Vincent J. Buonnano</li>
-            {% endif %}
-            {% endfor %}
-            {% endif %}
-            {% if print_mode %}
-            <ul><b>Print ID: </b>{{ print_id }}<a href="{{ studio_url }}" target="_blank"> [Library View]</a></ul>
-            {% if note != "no note" %}
-            <ul>{{note}}</ul>
-            {% endif %}
-            {% endif %}
-            <ul></ul>
-            <br />
-</div>
 
-{% if book_mode %}
+
+
 <div id="prevnext">
+    {% if book_mode %}
     <a id="prev_page">Loading...</a>
     <a id="next_page">Loading...</a>
+    {% endif %}
 </div>
-{% endif %}
+
+
 
 <div id="detailcontainer">
-<div id="image_span">
-    <iframe src={{ det_img_view_src}} width="100%" height="800" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
-</div>
+
+    <div id = "bookinfo">
+        {% if book_mode %}
+        <ul> <span style="color:#D64833">Book Information</span> <ul/><br />
+            {% endif %}
```

Comment:
can you fix the indenting here, please? The {% endif %} should be back inline with the {% if book_mode %} part. Then the {% if print_mode %} line that follows should be inline with the {% endif %}, and then the <ul>... line should be indented. Indenting things like that helps us understanding which if or for tag is active for a line a code.